hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/GeneralThrottleAccumulator.java

```diff
@@ -90,7 +91,7 @@ public class HandleThrottleAccumulator {
     private GasLimitDeterministicThrottle gasThrottle;
     private List<DeterministicThrottle> activeThrottles = Collections.emptyList();
 
-    public HandleThrottleAccumulator(@NonNull final ConfigProvider configProvider) {
+    public GeneralThrottleAccumulator(@NonNull final ConfigProvider configProvider) {
```

Comment:
I think we should name these classes differently. `GeneralThrottleAccumulator` and `HapiThrottling` provide almost the same functionality, but the names follow different patterns.

What do you think about calling this class `ThrottleAccumulator` and renaming `HapiThrottling` to `SynchronizedThrottleAccumulator`?